docs/website/docs/dlt-ecosystem/destinations/synapse.md

```diff
@@ -95,6 +95,16 @@ pipeline = dlt.pipeline(
     dataset_name='chess_data'
 )
 ```
+To use Active Directory Principal, you can use the `sqlalchemy.engine.URL.create` method to create the connection URL using your Active Directory Service Principal credentials. Once you have the connection URL, you can directly use it in your pipeline configuration or convert it to a string.
```

Comment:
@dat-a-man can you show an example in the code how sqlalchemy.engine.URL.create can be used ?

Reply:
@zem360: added one example